src/index.ts

```diff
@@ -361,6 +443,7 @@ export default class Wallet {
 
   // public instance methods
 
+  // tslint:disable-next-line
```

Comment:
Why is it necessary to ignore tslint errors here and in `getPublicKey()`?

Reply:
Without disabling those lines for tslint I get the following when doing `npm run tslint`:

```
ERROR: /Users/michael/repos/ethereumjs-wallet/src/index.ts:439:10 - Refactor this getter so that it actually refers to the property 'privateKey'
ERROR: /Users/michael/repos/ethereumjs-wallet/src/index.ts:448:10 - Refactor this getter so that it actually refers to the property 'publicKey'
```

I felt that refactoring that code from the [TS rewrite](https://github.com/ethereumjs/ethereumjs-wallet/pull/93) was beyond the scope of this PR, especially since the README documents those methods `get..` methods as part of the API. So I chose to stop the linter from failing on those lines.